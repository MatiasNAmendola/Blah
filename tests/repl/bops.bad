a: True
b: 42

False < 42
42 < (81 >= 52)
54 >= 54 >= a
