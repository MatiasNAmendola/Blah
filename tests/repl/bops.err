^<eval>
^<eval>
^<eval>
^<eval>
^<eval>
^<eval>
