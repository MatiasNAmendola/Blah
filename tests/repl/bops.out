<eval> comparison of "Vb False" and "Vi 42" is not supported
<eval> comparison of "Vi 42" and "Vb True" is not supported
<eval> comparison of "Vi 54" and "Vb True" is not supported
